Gemfile

```diff
@@ -30,3 +30,6 @@ gem "wdm", "~> 0.1.0" if Gem.win_platform?
 
 # Check if the html is valid
 gem "html-proofer", "~> 3.10.2"
+
+# Redirect plugin
+gem 'jekyll-redirect-from'
```

Comment:
Add the plugins in the plugins group

Reply:
The commented out part?

Reply:
Yup, it states with a comment above that the plugins should go there

Reply:
Oh ok lol I thought cause it was commented out that it wouldn't work hahaha